<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>208</width>
    <height>154</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Member</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>12</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Member ID:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>40</y>
     <width>63</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string> Node1 ID:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>70</y>
     <width>63</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string> Node2 ID:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="MemID_Input">
   <property name="geometry">
    <rect>
     <x>84</x>
     <y>12</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Node1ID_Input">
   <property name="geometry">
    <rect>
     <x>84</x>
     <y>40</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Node2ID_Input">
   <property name="geometry">
    <rect>
     <x>84</x>
     <y>70</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="OKButton">
   <property name="geometry">
    <rect>
     <x>116</x>
     <y>126</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>100</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Section ID:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SectionID_Input">
   <property name="geometry">
    <rect>
     <x>84</x>
     <y>100</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>MemID_Input</tabstop>
  <tabstop>Node1ID_Input</tabstop>
  <tabstop>Node2ID_Input</tabstop>
  <tabstop>SectionID_Input</tabstop>
  <tabstop>OKButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
